packages/block-editor/src/components/block-edit/edit.js

```diff
@@ -120,7 +133,10 @@ const EditWithGeneratedProps = ( props ) => {
 			) ) {
 				const { source: sourceName, args: sourceArgs } = binding;
 				const source = registeredSources[ sourceName ];
-				if ( ! source || ! canBindAttribute( name, attributeName ) ) {
+				if (
+					! source ||
+					! bindableAttributes.includes( attributeName )
```

Comment:
Getting errors in the site editor 
<img width="476" height="145" alt="image" src="https://github.com/user-attachments/assets/068cd137-6744-4d07-b346-9f06702e1df8" />

Reply:
Fix: https://github.com/WordPress/gutenberg/pull/72401
Revert: https://github.com/WordPress/gutenberg/pull/72400